anime-list-master.xml

```diff
@@ -53,6 +53,12 @@
   </anime>
   <anime anidbid="12" tvdbid="72070" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Chobits</name>
+	<mapping-list>
+	  <mapping anidbseason="0" tvdbseason="0">;1-3;2-4;</mapping>
+	  <mapping anidbseason="1" tvdbseason="1" start="10" end="17" offset="-1"/>
+	  <mapping anidbseason="1" tvdbseason="1" start="18" end="26" offset="-2"/>
+	  <mapping anidbseason="1" tvdbseason="0">;9-1;10-2;</mapping>
```

Comment:
Shouldn't the second one be 18 instead of 10?

Reply:
Ah crap, yea that was a typo. Don't know how i missed that.